python/tools/whisper/Dockerfile.dockerignore

```
@@ -0,0 +1,2 @@
+*
```

Hadolint (2.14.0)

[error] unexpected '*'
expecting '#', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, end of input, or whitespaces

(DL1000)